sdk/webpubsub/azure-messaging-webpubsub/src/main/java/com/azure/messaging/webpubsub/models/GetAuthenticationTokenOptions.java

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.messaging.webpubsub.models;
+
+import java.time.Duration;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import com.azure.messaging.webpubsub.WebPubSubAsyncServiceClient;
+import com.azure.messaging.webpubsub.WebPubSubServiceClient;
+
+/**
+ * Options class for configuring the
+ * {@link WebPubSubAsyncServiceClient#getAuthenticationToken(GetAuthenticationTokenOptions)} and
+ * {@link WebPubSubServiceClient#getAuthenticationToken(GetAuthenticationTokenOptions)} methods.
+ */
+public final class GetAuthenticationTokenOptions {
+    private Duration expiresAfter;
```

Comment:
Should this be final?

Reply:
`expiresAfter` can be updated through the setter method below.